grpc/tests/grpctest.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import print_function
+
+import os
+import sys
+import grpc
+import flatbuffers
+
+from concurrent import futures
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..', 'tests'))
+import MyGame.Example.Monster as Monster
+import MyGame.Example.Stat as Stat
+import MyGame.Example.monster_test_grpc_fb as monster_grpc_fb
+
+test_stat_id = "test_stat_id"
+test_monster_name = "test_monster_name"
+
+
+class MonsterStorage(monster_grpc_fb.MonsterStorageServicer):
+
+    def Store(self, request, context):
+
+        m = Monster.Monster().GetRootAsMonster(request, 0)
+
+        b = flatbuffers.Builder(0)
+        i = b.CreateString(test_stat_id)
+        Stat.StatStart(b)
+        Stat.StatAddId(b, i)
+        b.Finish(Stat.StatEnd(b))
+        return bytes(b.Output())
+
+    def Retrieve(self, request, context):
+
+        s = Stat.Stat().GetRootAsStat(request, 0)
+
+        b = flatbuffers.Builder(0)
+        i = b.CreateString(test_monster_name)
+        Monster.MonsterStart(b)
+        Monster.MonsterAddName(b, i)
+        b.Finish(Monster.MonsterEnd(b))
+        yield bytes(b.Output())
+
+
+def serve():
+
+    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+    monster_grpc_fb.add_MonsterStorageServicer_to_server(MonsterStorage(), server)
+    server.add_insecure_port('[::]:50051')
+    server.start()
+    run()
+
+
+def run():
+
+    channel = grpc.insecure_channel('127.0.0.1:50051')
+    stub = monster_grpc_fb.MonsterStorageStub(channel)
+
+    b = flatbuffers.Builder(0)
```

Comment:
Could you run the full Monster tests here? That way we can exercise more than just a few generated functions.

Reply:
I have updated my tests. Is this what you had in mind @rw?

Reply:
@bsoniam Would you be able to use `CheckReadBuffer` (or something like it that is comprehensive) here? That's what I mean. Source: https://github.com/google/flatbuffers/blob/master/tests/py_test.py#L80